pkgs/development/libraries/dpp/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, libopus
+, openssl
+, zlib
+, libsodium
+, gcc
+, pkg-config
+, autoPatchelfHook
+, lib
+}:
+let
+   libExt = if stdenv.hostPlatform.isDarwin then ".dylib" else ".so";
+in
+stdenv.mkDerivation rec {
+  pname = "dpp";
+  version = "10.0.29";
+
+  src = fetchFromGitHub {
+    owner = "brainboxdotcc";
+    repo = "DPP";
+    rev = "v${version}";
+    sha256 = "BJMg3MLSfb9x/2lPHITeI3SWwW1OZVUUMVltTWUcw9I=";
+  };
+
+  nativeBuildInputs =  [
+    cmake
+    libopus
+  ] ++ lib.optional (stdenv.hostPlatform.isLinux) autoPatchelfHook;
+
+  buildInputs = [
+    openssl
+    zlib
+    libsodium
+    libopus
+    gcc
+    pkg-config
+  ];
+
+  PKG_CONFIG_PATH = [
+    "${openssl.dev}/lib/pkgconfig"
+    "${libsodium.dev}/lib/pkgconfig"
+    "${libopus.dev}/lib/pkgconfig"
+  ];
+
+  cmakeFlags = [
+    "-DLibsodium_INCLUDE_DIR=${libsodium.dev}/include"
+    "-DLibsodium_LIBRARY=${libsodium.out}/lib"
+    "-DOPENSSL_ROOT_DIR=${openssl.dev}"
+    "-DOPENSSL_LIBRARIES=${openssl.out}/lib"
+    "-DOPENSSL_CRYPTO_LIBRARY=${openssl.out}/lib/libcrypto${libExt}"
+    "-DOPENSSL_SSL_LIBRARY=${openssl.out}/lib/libssl${libExt}"
+    "-DOPENSSL_INCLUDE_DIR=${openssl.dev}/include"
+  ];
+
+  meta = {
+    description = "D++ Discord C++ Library";
+    homepage = "https://github.com/brainboxdotcc/DPP";
+    license = lib.licenses.asl20;
+    maintainers = with lib.maintainers; [ renato ];
```

Comment:
This should be you.

Reply:
I changed it to  rgchavezchicoma is that alright